missions/ULM_missions.txt

```diff
@@ -0,0 +1,335 @@
+ULM_diplomacy = {
+	slot = 1
+	generic = no
+	ai = yes
+	potential = {
+		tag = ULM
+	}
+	has_country_shield = yes
+
+
+	ULM_hre_allies = {
+		icon = mission_alliances.png
+		required_missions = { }
+		position = 1
+		
+		trigger = {
+			calc_true_if = { # alejandro = god
+				amount = 2
+				all_countries = {
+					is_part_of_hre = yes
+					allied_to = ROOT
+				}
+			}
+		}
+
+		effect = {
+			prestige = 20
```

Comment:
Should be
```add_prestige = 20
```
prestige is a trigger, not an effect.

Reply:
not sure why but i only see a black box for your comment, but yeh that should be "add_prestige"